.eslintrc.cjs

```diff
@@ -12,17 +12,33 @@ module.exports = {
 	overrides: [
 		{
 			extends: ["plugin:markdown/recommended"],
-			files: ["**/*.{md}"],
+			files: ["**/*.md", "**/*.md/*.js"],
```

Comment:
[Question] This I don't follow. Why use the Markdown parser on `.js` snippets? Wouldn't we want this to be the same as any `.cjs`/`.js` file?

Reply:
I thought I need to add this explicitly so it will disable some problematic rules, turns out it actually broke linting :see_no_evil: 

https://github.com/eslint/eslint-plugin-markdown#frequently-disabled-rules

Looks like it's working on js files now

![Screenshot_2023-03-12_12-43-12](https://user-images.githubusercontent.com/12304307/224543332-50519ea1-e11c-4f06-bf88-171c30545176.png)